pytest.ini

```diff
@@ -1,2 +1,3 @@
 [pytest]
 filterwarnings = ignore::urllib3.exceptions.InsecureRequestWarning
+addopts = --driver=Chrome
```

Comment:
This option should be added when running pytest. Automation is not doing any UI testing right now and passing this option by default will break on system where the infrastructure to run UI tests are not present (for example our jenkins)

Reply:
will move this to makefile args.